samples/javascript_nodejs/09.message-routing/README.md

```diff
@@ -37,6 +36,29 @@ This samples shows how to:
 - File -> Open Bot Configuration and navigate to `javascript_nodejs/09.message-routing` folder
 - Select `message-routing.bot` file
 
+# Deploy this bot to Azure
+To clone this bot, perform the following:
+- Collect your Luis Authoring Key from the the [LUIS portal](https://www.luis.ai) by selecting your name in the top right corner. Save this key for the next step.
+
+- Run the following command from the project directory:
+```bash
+msbot clone services --name "<NAME>" --folder deploymentScripts/msbotClone --location "e.g, westus"
+```
+**NOTE**: By default your Luis Applications will be deployed to your free starter endpoint. An Azure LUIS service will be deployed along with your bot but you must manually add and publish to it from the luis.ai portal and update your key in the .bot file.
+
+-  Note the generated secret generated by MSBot. 
+- The secret key is used later in for the emulator and configuration.
```

Comment:
Maybe you meant either `The secret key is used later in the emulator` or `The secret key is used later for the emulator`?

Reply:
Solved @VictorGrycuk 
Note that this change has been made by vishwacsena, probably it has been carried out to other samples using this section.